src/interfaces/store-interfaces/state-interface.ts

```diff
@@ -6,6 +6,7 @@ export interface StateModelIF {
   tombstone: TombStoneIF
   accountInformation: AccountInformationIF
   nameRequest: NameRequestIF
+  nameTranslations?: Array<string>
```

Comment:
This always exists in state, doesn't it? So doesn't need to be optional.

Reply:
ah, thanks! Will update in next commit